src/main/java/org/folio/consortia/messaging/listener/ConsortiaEventListener.java

```diff
@@ -55,8 +58,7 @@ public String getCentralTenantByIdByHeader(MessageHeaders messageHeaders) {
     String centralTenantId;
 
     // getting central tenant for this requested tenant from get central in its own schema
-    try (var context = new FolioExecutionContextSetter(createFolioExecutionContext(
-      messageHeaders, folioMetadata, requestedTenantId))) {
+    try (var context = new FolioExecutionContextSetter(contextHelper.getFolioExecutionContext(requestedTenantId))) {
```

Comment:
I am not sure if we need to use system user for processing kafka messages because it will slow down this processing, I propose to start using system user only in TenantService when initially setting the tenant for now.

Reply:
Fixed